modules/installation-gcp-user-infra-completing.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-infra-installation_{context}"]
+= Completing a GCP installation on user-provisioned infrastructure
+
+After you start the {product-title} installation on Google Cloud Platform (GCP)
+user-provisioned infrastructure, you can monitor the cluster events until the
+cluster is ready.
+
+.Prerequisites
+
+* Deploy the bootstrap machine for an {product-title} cluster on user-provisioned GCP infrastructure.
+* Install the `oc` CLI and log in.
+
+.Procedure
+
+. Complete the cluster installation:
++
+----
+$ ./openshift-install --dir=<installation_directory> wait-for install-complete <1>
+
+INFO Waiting up to 30m0s for the cluster to initialize...
+----
+<1> For `<installation_directory>`, specify the path to the directory that you
+stored the installation files in.
+
+. Run the following commands to observe the running state of your cluster pods:
```

Comment:
Pods

I'd split this command up a bit - the three commands are getting lost in the output, and I'd explain the difference in the information that you're getting.